### ISSUE TITLE
General build of Experience Sampler app?

### ISSUE THREAD

**Issue by Zendaug** [sabrinathai/ExperienceSampler#37]
Hi folks,

I really love the work that you've done with Experience Sampler. A quick question -- are there any general builds of the Experience Sampler already available for Android/IOS that could be customised with new questions and/or schedules? As I understand it, the general approach for Experience Sampler is to hard code the content of the survey into the app, and then distribute it. However, it would certainly make the distribution of the app easier if an approved version already existed in the App Store / Google Play Store that could be populated with new content.

Best wishes,

- Tim

**Comment by sabrinathai** [comment]
Hi Tim,

Sadly, there is no approved version that already exists on the App Store. The App Store rejected our app because it did not meet the requirements.

**Comment by brittanyreese** [comment]
Does this mean that an app built with Experience Sampler has yet to be published to the App Store?

**Comment by sabrinathai** [comment]
That is correct. No ExperienceSampler apps have been published to the App Store.
